src/java/io/bazel/rulesscala/scalac/ScalacProcessor.java

```diff
@@ -247,6 +248,13 @@ private static void compileScalaSources(CompileOptions ops, String[] scalaSource
       System.err.println("Compiler runtime: " + (stop - start) + "ms.");
     }
 
+    try {
+      PrintWriter writer = new PrintWriter(ops.statsfile, "UTF-8");
+      writer.println(Long.toString(stop - start));
+      writer.close();
+    } catch (Throwable ex) {
```

Comment:
catching throwable is a bad idea because it catches JVM errors
you need to catch IOException, right?